scripts/startLocalCluster.sh

```diff
@@ -179,6 +212,29 @@ start() {
     mkdir -p cluster/data$PORT cluster/apps$PORT
     echo == Starting $TYPE on port $PORT
     [ "$INTERACTIVE_MODE" == "R" ] && sleep 1
+    if [ "$AUTOUPGRADE" == "1" ];then
+      $CMD \
+          -c none \
+          --database.directory cluster/data$PORT \
+          --cluster.agency-endpoint $TRANSPORT://$ENDPOINT:$AG_BASE \
+          --cluster.my-address $TRANSPORT://$ADDRESS:$PORT \
+          --server.endpoint $TRANSPORT://$ENDPOINT:$PORT \
+          --cluster.my-role $ROLE \
+          --log.file cluster/$PORT.log \
+          --log.level $LOG_LEVEL \
+          --server.statistics true \
+          --javascript.startup-directory $SRC_DIR/js \
+          --javascript.module-directory $SRC_DIR/enterprise/js \
+          --javascript.app-path cluster/apps$PORT \
+          --log.force-direct true \
+          --log.level $LOG_LEVEL_CLUSTER \
+          --javascript.allow-admin-execute true \
+          $STORAGE_ENGINE \
+          $AUTHENTICATION \
+          $SSLKEYFILE \
+          --database.auto-upgrade true
```

Comment:
is this missing the backslash at the end of the line?

Reply:
Yes, will fix.